src/actions/dialog.js

```diff
@@ -0,0 +1,304 @@
+export function updateVmDialogErrorMessage (message) {
```

Comment:
Please rename this file to something more descriptive and related to its purpose.